src/Yoti/Http/Request.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+namespace Yoti\Http;
+
+use Yoti\Exception\RequestException;
+
+class Request
```

Comment:
We've called this _Request_, but it doesn't actually represent a request object.
Can we call it _RequestSender_, or something more appropriate to what it does?

Reply:
What's a request Object in your opinion?
Because to me it's a Request Object.